xpring-eng/rippled-network-crawler
# Feat: Sets option to teardown db through config

## High Level Overview of Change
- Makes resetting db configurable

### Context of Change
We were previously manually doing resets of the dbs in the dockerfile branch. This PR moves the option to do this to the config so that we can set it in the explorer repo during deployments. This will allow us to merge the dockerfile branch (the branch we were deploying for) into master and deploy (from explorer repo) crawler directly from master.

We do this by setting the reset_db variable in config (which defaults to false). We will need to coordinate multiple PRs in crawler and explorer to make this change take effect.

### Type of Change


## Before / After
Now you need to set `reset_db` for `node setup` to clear dbs.

## Test Plan
1. Set `reset_db` to false, run node setup, nothing should happen
2. Set `reset_db` to true, run node setup, previous db should be dropped and re init
